### PR TITLE
Sous Chefs Adoption

### PR DESCRIPTION
**Pull request by EMH333**
Adopt cookbook into Sous Chefs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ This file is used to list changes made in each version of the openssh cookbook.
 
 ## Unreleased
 
+- Sous Chefs Adoption
+- Cookstyle fixes
+- Standardise files with files in sous-chefs/repo-management
 - resolved cookstyle error: libraries/helpers.rb:81:16 warning: `Lint/SendWithMixinArgument`
```
Comment (ramereth) [inline CHANGELOG.md:10]: I've been rewriting these in a condensed form and just have `Cookstyle fixes` to help with how this gets displayed in the announcement channel.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # openssh Cookbook
 
-[![Build Status](https://travis-ci.org/chef-cookbooks/openssh.svg?branch=master)](https://travis-ci.org/chef-cookbooks/openssh) [![Cookbook Version](https://img.shields.io/cookbook/v/openssh.svg)](https://supermarket.chef.io/cookbooks/openssh)
+[![Cookbook Version](https://img.shields.io/cookbook/v/openssh.svg)](https://supermarket.chef.io/cookbooks/openssh)
+[![CI State](https://github.com/sous-chefs/openssh/workflows/ci/badge.svg)](https://github.com/sous-chefs/openssh/actions?query=workflow%3Aci)
+[![OpenCollective](https://opencollective.com/sous-chefs/backers/badge.svg)](#backers)
+[![OpenCollective](https://opencollective.com/sous-chefs/sponsors/badge.svg)](#sponsors)
+[![License](https://img.shields.io/badge/License-Apache%202.0-green.svg)](https://opensource.org/licenses/Apache-2.0)
 
 Installs and configures OpenSSH client and daemon.
```
Comment (ramereth) [inline README.md:10]: Please add the following before ``## Requirements``:
```
## Maintainers

This cookbook is maintained by the Sous Chefs. The Sous Chefs are a community of Chef cookbook maintainers working together to maintain important cookbooks. If you’d like to know more please visit [sous-chefs.org](https://sous-chefs.org/) or come chat with us on the Chef Community Slack in [#sous-chefs](https://chefcommunity.slack.com/messages/C2V7B88SF).
```

**Review by ramereth — CHANGES_REQUESTED** [review]
Also, please go ahead and remove Ubuntu 16.04 support/testing.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      - name: Dokken
+        uses: actionshub/kitchen-dokken@main
+        env:
+          CHEF_LICENSE: accept-no-persist
+          KITCHEN_LOCAL_YAML: kitchen.dokken.yml
+        with:
+          suite: ${{ matrix.suite }}
+          os: ${{ matrix.os }}
+
+# unable to get SSH service to start
```
Comment (ramereth) [inline .github/workflows/ci.yml:78]: Can you please deal with the check warning here from yamllint?

File: CHANGELOG.md (hunk 1 above)
Comment (ramereth) [reply CHANGELOG.md:8]: Let me clarify, I **replace** the automated messages with *just* `Cookstyle fixes`. Can you please make that change here?

File: kitchen.yml [hunk 4]
```diff
@@ -3,46 +3,43 @@ driver:
...
-    driver_config:
-      box: mvbcoding/awslinux
   - name: amazonlinux-2
-  - name: centos-6
   - name: centos-7
   - name: centos-8
-  - name: debian-8
-    run_list: apt::default
   - name: debian-9
     run_list: apt::default
```
Comment (ramereth) [inline kitchen.yml:19]: This should be removed from any debian-based host in the regular kitchen.yml file.

**Review by ramereth — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (EMH333) [reply CHANGELOG.md]: ah, derp

**Review by ramereth — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [2.9.0](https://github.com/sous-chefs/openssh/releases/tag/2.9.0)

**Pull request merged**